package.json

```diff
@@ -3626,10 +3626,10 @@
 		"watch-non-ext": "tsc -p ./tsconfig.build.json --watch --extendedDiagnostics",
 		"lint-fix": "eslint . --fix",
 		"lint": "eslint .",
-		"test": "npm run build && npm run build-tests && npm run instrument-dist && npm run instrument && npm run test-only && npm run report_lcov && npm run report_screen",
+		"test": "sh -c 'npm run build && npm run build-tests && npm run instrument-dist && npm run instrument && npm run test-only \"$@\" && npm run report_lcov && npm run report_screen' --",
```

Comment:
The changes must work on all platforms, including Windows. Will this work there?